pkg/clusteragent/admission/mutate/autoinstrumentation/target_mutator.go

```diff
@@ -215,10 +240,21 @@ func (m *TargetMutator) MutatePod(pod *corev1.Pod, ns string, _ dynamic.Interfac
 // ShouldMutatePod determines if a pod would be mutated by the target mutator. It is used by other webhook mutators as
 // a filter.
 func (m *TargetMutator) ShouldMutatePod(pod *corev1.Pod) bool {
-	// Return if the mutator is disabled.
+	enabledLabelVal, enabledLabelExists := getEnabledLabel(pod)
```

Comment:
Add a comment clarifying that we are explicitly looking for "opt-out" of mutation here which is why we only check for the negation.